package.json

```
@@ -96,9 +96,12 @@
   },
   "eslintConfig": {
     "root": true,
-    "extends": ["react-app"],
+    "extends": [
+      "react-app"
+    ],
```

It turns out we don't need the array:
```suggestion
    "extends": "react-app",
```

---

Yeah, but I think I'm gonna keep it because I plan to add more later.